src/runner/browser-env/vite/browser-modules/constants.ts

```diff
@@ -12,3 +12,9 @@ export const VITE_SELECTORS = {
 
 export const BROWSER_EVENT_SUFFIX = "browser";
 export const WORKER_EVENT_SUFFIX = "worker";
+
+// TODO: use from nodejs code after migrate to esm
+export const SOCKET_MAX_TIMEOUT = 2147483647;
```

Comment:
All emitted events with using `emitWithAck` on server side (in testplane master process) must set timeout, which cannot be specifies as zero or `Infinity`. This is max value which accepts socket.io